README.md

```diff
@@ -1,2 +1,5 @@
-# katacoda-scenarios
-kubesphere Katacoda Scenarios
+# Interactive kubesphere Katacoda Scenarios
+[![](http://shields.katacoda.com/katacoda/kubesphere-sigs/count.svg)](https://www.katacoda.com/kubesphere-sigs "Get your profile on Katacoda.com")
+
+Visit https://www.katacoda.com/kubesphere-sigs to view the profile and interactive scenarios
```

Comment:
![image](https://user-images.githubusercontent.com/1450685/131427274-174fffd0-64a8-4e3d-924b-0999d8dbcc02.png)

Reply:
I think we should cretae a profile for the organization in katacoda 
Similar to what i did in my account

Reply:
Got you. Please go ahead, and help us to create a profile.